test failure on 3.6.1

Hello,

while emerging pg_activity-3.6.1 on gentoo I get the following test failure:

```
python3.12 -m pytest -vv -ra -l -Wdefault --color=yes -o console_output_style=count -o tmp_path_retention_count=0 -o tmp_path_retention_policy=failed --junit-xml=/var/tmp/portage/dev-db/pg_activity-3.6.1-r2/temp/pytest-xml/python3.12-BZlr9U.xml -o junit_family=xunit1 --force-short-summary -p no:cov -p no:flake8 -p no:flakes -p no:pylint -p no:markdown -p no:sugar -p no:xvfb -p no:pytest-describe -p no:plus -p no:tavern -p no:salt-factories -k not test_ui.txt and not test_data.py and not test_cli_help.txt and not test_cli_help_py312.txt
============================= test session starts ==============================
platform linux -- Python 3.12.13, pytest-9.0.2, pluggy-1.6.0 -- /var/tmp/portage/dev-db/pg_activity-3.6.1-r2/work/pg_activity-3.6.1-python3_12/install/usr/bin/python3.12
cachedir: .pytest_cache
psycopg: 3.3.3
rootdir: /var/tmp/portage/dev-db/pg_activity-3.6.1-r2/work/pg_activity-3.6.1
configfile: pytest.ini
collecting ... collected 80 items / 19 deselected / 61 selected

pgactivity/activities.py::pgactivity.activities.sorted PASSED           [ 1/61]
pgactivity/activities.py::pgactivity.activities.update_max_iops PASSED  [ 2/61]
pgactivity/config.py::pgactivity.config.Configuration.parse PASSED      [ 3/61]
pgactivity/config.py::pgactivity.config.Flag PASSED                     [ 4/61]
pgactivity/data.py::pgactivity.data.Data.get_duration_column PASSED     [ 5/61]
pgactivity/data.py::pgactivity.data.pg_get_short_version PASSED         [ 6/61]
pgactivity/handlers.py::pgactivity.handlers.duration_mode PASSED        [ 7/61]
pgactivity/handlers.py::pgactivity.handlers.query_mode PASSED           [ 8/61]
pgactivity/handlers.py::pgactivity.handlers.refresh_time PASSED         [ 9/61]
pgactivity/handlers.py::pgactivity.handlers.sort_key_for PASSED         [10/61]
pgactivity/handlers.py::pgactivity.handlers.wrap_query PASSED           [11/61]
pgactivity/keys.py::pgactivity.keys._sequence_by_int PASSED             [12/61]
pgactivity/types.py::pgactivity.types.Column PASSED                     [13/61]
pgactivity/types.py::pgactivity.types.SelectableProcesses PASSED        [14/61]
pgactivity/types.py::pgactivity.types.SystemInfo.default PASSED         [15/61]
pgactivity/types.py::pgactivity.types.UI.column PASSED                  [16/61]
pgactivity/types.py::pgactivity.types.UI.columns PASSED                 [17/61]
pgactivity/types.py::pgactivity.types.UI.end_interactive PASSED         [18/61]
pgactivity/types.py::pgactivity.types.UI.evolve PASSED                  [19/61]
pgactivity/types.py::pgactivity.types.UI.start_interactive PASSED       [20/61]
pgactivity/types.py::pgactivity.types.UI.tick_interactive PASSED        [21/61]
pgactivity/types.py::pgactivity.types.UI.toggle_pause PASSED            [22/61]
pgactivity/types.py::pgactivity.types.UIHeader.make PASSED              [23/61]
pgactivity/types.py::pgactivity.types.UIHeader.toggle_instance PASSED   [24/61]
pgactivity/types.py::pgactivity.types.UIHeader.toggle_system PASSED     [25/61]
pgactivity/types.py::pgactivity.types.UIHeader.toggle_workers PASSED    [26/61]
pgactivity/types.py::pgactivity.types.enum_next PASSED                  [27/61]
pgactivity/utils.py::pgactivity.utils.MessagePile PASSED                [28/61]
pgactivity/utils.py::pgactivity.utils.clean_str PASSED                  [29/61]
pgactivity/utils.py::pgactivity.utils.csv_write PASSED                  [30/61]
pgactivity/utils.py::pgactivity.utils.ellipsis PASSED                   [31/61]
pgactivity/utils.py::pgactivity.utils.format_duration PASSED            [32/61]
pgactivity/utils.py::pgactivity.utils.get_duration PASSED               [33/61]
pgactivity/utils.py::pgactivity.utils.short_state PASSED                [34/61]
pgactivity/utils.py::pgactivity.utils.wait_status PASSED                [35/61]
pgactivity/utils.py::pgactivity.utils.yn PASSED                         [36/61]
pgactivity/views.py::pgactivity.views.format_query PASSED               [37/61]
pgactivity/views.py::pgactivity.views.get_indent PASSED                 [38/61]
pgactivity/views.py::pgactivity.views.limit PASSED                      [39/61]
pgactivity/views.py::pgactivity.views.query_mode PASSED                 [40/61]
pgactivity/views.py::pgactivity.views.shorten FAILED                    [41/61]
tests/test_activities.py::test_ps_complete PASSED                       [42/61]
tests/test_activities.py::test_ps_complete_empty_procs PASSED           [43/61]
tests/test_activities.py::test_mem_swap_load PASSED                     [44/61]
tests/test_cli.py::test_parser PASSED                                   [45/61]
tests/test_cli.py::test_parser_flag_on PASSED                           [46/61]
tests/test_config.py::test_flag PASSED                                  [47/61]
tests/test_config.py::test_flag_load PASSED                             [48/61]
tests/test_config.py::test_error PASSED                                 [49/61]
tests/test_config.py::test_lookup PASSED                                [50/61]
tests/test_config.py::test_lookup_builtin_profiles[minimal-expected0] PASSED [51/61]
tests/test_config.py::test_lookup_builtin_profiles[narrow-expected1] PASSED [52/61]
tests/test_config.py::test_lookup_builtin_profiles[wide-expected2] PASSED [53/61]
tests/test_scroll.txt::test_scroll.txt PASSED                           [54/61]
tests/test_types.py::test_filters_from_options PASSED                   [55/61]
tests/test_views.py::test_columns_header[pid+database; sort by cpu; activities] PASSED [56/61]
tests/test_views.py::test_columns_header[cpu; sort by cpu; activities] PASSED [57/61]
tests/test_views.py::test_columns_header[mem; sort by cpu; activities] PASSED [58/61]
tests/test_views.py::test_columns_header[many flags; sort by duration; blocking] PASSED [59/61]
tests/test_views.txt::test_views.txt FAILED                             [60/61]
tests/test_widgets.txt::test_widgets.txt PASSED                         [61/61]

=================================== FAILURES ===================================
______________________ [doctest] pgactivity.views.shorten ______________________
EXAMPLE LOCATION UNKNOWN, not showing all tests of that example
??? >>> shorten(term, text, 6)
Expected:
    'hello,'
Got:
    '[...]'

/var/tmp/portage/dev-db/pg_activity-3.6.1-r2/work/pg_activity-3.6.1/pgactivity/views.py:None: DocTestFailure
___________________________ [doctest] test_views.txt ___________________________
540 <BLANKLINE>
541 <BLANKLINE>
542 <BLANKLINE>
543 <BLANKLINE>
544 <BLANKLINE>
545 <BLANKLINE>
546 F1/1 Running queries    F2/2 Waiting queries    F3/3 Blocking queries   Space Pause/unpause     q Quit                  h Help                        
547 
548 (using terminal width)
549 >>> screen(
Differences (unified diff with -expected +actual):
    @@ -1,8 +1,8 @@
    -PostgreSQL 13 - localhost - tester@host:5432/postgres - Ref.: 10s - Duration
    - * Global: 132 days, 3 hours and 21 minutes uptime, 19.53M dbs size - 1.00K/s
    -   Sessions: 6/100 total, 1 active, 1 idle, 10 idle in txn, 0 idle in txn abrt,
    -   Activity: 15 tps, 10 insert/s, 20 update/s, 30 delete/s, 40 tuples
    +PostgreSQL 13 - localhost - tester@host:5432/postgres - Ref.: 10s - [...]
    + * Global: 132 days, 3 hours and 21 minutes uptime, 19.53M dbs size - [...]
    +   Sessions: 6/100 total, 1 active, 1 idle, 10 idle in txn, 0 idle in txn [...]
    +   Activity: 15 tps, 10 insert/s, 20 update/s, 30 delete/s, 40 tuples [...]
      * Worker processes: 3/8 total, 1/8 logical workers, 2/8 parallel workers
    -   Other processes & info: 3/3 autovacuum workers, 1/10 wal senders, 0 wal
    +   Other processes & info: 3/3 autovacuum workers, 1/10 wal senders, 0 wal [...]
      * Mem.: 8B total, 2B (25.00%) free, 2B (25.00%) used, 4B (50.00%) buff+cached
        Swap: 2B total, 1B (50.00%) free, 1B (50.00%) used

/var/tmp/portage/dev-db/pg_activity-3.6.1-r2/work/pg_activity-3.6.1/tests/test_views.txt:549: DocTestFailure
- generated xml file: /var/tmp/portage/dev-db/pg_activity-3.6.1-r2/temp/pytest-xml/python3.12-BZlr9U.xml -
=========================== short test summary info ============================
FAILED pgactivity/views.py::pgactivity.views.shorten
FAILED tests/test_views.txt::test_views.txt
================= 2 failed, 59 passed, 19 deselected in 0.22s ==================
```

I'm pretty sure that in the past the tests passed, see: https://github.com/dalibo/pg_activity/issues/435#issuecomment-2935347826
So the failure may depend from an external python module

If there is something else I can help, please let me know.
After a fix, if you can make a new minor release, would be great.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -7,13 +7,15 @@
 * Strip comments from SQL queries emitted by pg\_activity (#442).
 * Remove the "tuples returned/s" metric from the header, it provided misleading 
   information (#441, reported by @kmoppel)
+* Fix a difference in wrapping behavior introduced in blessed 1.26 (#455,
+  reported by @asarubbo)
 
 ## pg\_activity 3.6.1 - 2025-06-03
 
 ### Fixed
 
-*  Fix `--hide-queries-in-logs` to also disable log_statements when it is used
-   (Kouber Saparev).
+* Fix `--hide-queries-in-logs` to also disable log_statements when it is used
+  (Kouber Saparev).
 * Fix deprecated syntax of the `license` field in packaging metadata; require
   setuptools version 77.0.0 or higher accordingly.
 * Remove dummy `setup.py` file.

--- a/pgactivity/views.py
+++ b/pgactivity/views.py
@@ -129,12 +129,10 @@ def limit(func: Callable[..., Iterable[str]]) -> Callable[..., None]:
 def help(term: Terminal, version: str, is_local: bool) -> Iterable[str]:
     """Render help menu."""
     project_url = "https://github.com/dalibo/pg_activity"
-    intro = dedent(
-        f"""\
+    intro = dedent(f"""\
     {term.bold_green}pg_activity {version} - {link(term, project_url, project_url)}
     {term.normal}Released under PostgreSQL License.
-    """
-    )
+    """)
 
     def key_mappings(keys: Iterable[Key]) -> Iterable[str]:
         for key in keys:

--- a/pgactivity/views.py
+++ b/pgactivity/views.py
@@ -71,7 +71,7 @@ def shorten(term: Terminal, text: str, width: int | None = None) -> str:
     """
     if not text:
         return ""
-    wrapped = term.wrap(text, width=width, max_lines=1)
+    wrapped = term.wrap(text, width=width, max_lines=1, placeholder="")
     return wrapped[0] + term.normal
 
 

